go/labels/labels.go

```diff
@@ -44,6 +44,9 @@ const (
 	TaskTypeDerivation = "derivation"
 	// TaskTypeMaterialization is a "materialization" TaskType.
 	TaskTypeMaterialization = "materialization"
+	// CatalogRevision is a base-10 integer of the minimum ETCD revision that must be observed
+	// before this catalog task can be started.
+	CatalogRevision = "estuary.dev/catalog-revision"
```

Comment:
`estuary.dev/task-created` ? "Catalog" isn't a concrete thing (it's CatalogTasks or the CatalogCommons).

Reply:
Yeah I was being intentionally vague with the definition regarding exactly what this revision corresponds to. Technically, the revision is just whatever is returned from running the etcd transaction that applies the catalog tasks _and_ commons, so I didn't want to assume or imply that it would correspond exactly to the revision of the commons or any specific task. So the meaning I was going for is that you need to have observed up through this revision before you can be guaranteed to observe the updated task and commons. That said, I think `task-created` is a perfectly reasonable name, as long as we clarify that it doesn't correspond exactly to the revision of the specific task.